packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.js

```diff
@@ -128,11 +129,16 @@ export default class QuanticRefineToggle extends LightningElement {
   };
 
   get refineButtonDisabled() {
-    const areFacetsRendered = Object.values(this.renderedFacets).reduce(
+    return (
+      (!this.hasResults && !this.activeFiltersCount) || this.isContentEmpty
```

Comment:
```suggestion
      !(this.hasResults || this.activeFiltersCount) || this.isContentEmpty
```
Do you prefer this?

Reply:
Extracting variables makes it easier to read, I think. However, I have trouble finding a good name for the `noResults && noFacetsSelected` part, which is probably why it is hard to read in the first place.

```suggestion
const noResults = !this.hasResults;
const noFacetsSelected = !this.activeFiltersCount;
return (noResults && noFacetsSelected) || this.isContentEmpty;
```